src/App.js

```diff
@@ -19,6 +19,8 @@ const App = () => {
     WebViewer(
       {
         path: '/webviewer/lib',
+        enableRedaction: process.env.DEMO,
+        disableVirtualDisplayMode: true,
```

Comment:
`disableVirtualDisplayMode: true` why did you add this back? I removed it in a commit because I don't think we need it.